Objects/stringlib/unicode_format.h

```diff
@@ -440,8 +440,13 @@ get_field_object(SubString *input, PyObject *args, PyObject *kwargs,
 
         /* look up in args */
         obj = PySequence_GetItem(args, index);
-        if (obj == NULL)
-            goto error;
+        if (obj == NULL) {
+            PyErr_Format(PyExc_IndexError,
+                         "Replacement index %d out of range for positional "
```

Comment:
I believe `%zd` should be used for `Py_ssize_t` (commit d1469449092563d375feb6e3ca67d00e9e638403 recently fixed some improper uses of `%d`).